dispatcher/nginx.conf.template

```diff
@@ -117,6 +117,7 @@ http {
             proxy_set_header X-Real-IP $remote_addr;
             proxy_pass http://webui;
             proxy_redirect http:// $scheme://;
+            proxy_set_header "mTLS-DN" '$ssl_client_s_dn';
```

Comment:
should we, maybe, put this under some "if afc_mtls_enforce" But be careful, ifs in  nginx configs are very tricky

Reply:
resolved in a call - no problems during testing